source/elements/oneDPL/source/parallel_api.rst

```diff
@@ -262,6 +262,18 @@ Iterators
 The oneDPL iterators are defined in the :code:`<oneapi/dpl/iterator>` header,
 in :code:`namespace oneapi::dpl`.
 
+Let us define a named requirement, :code:`ValidParallelIteratorSource`, to describe valid random access iterator-like
+types that can be used as source for oneDPL's iterators as described below.
+The type :code:`Iter` satisfies the :code:`ValidParallelIteratorSource` named requirement if it satisfies
+at least one of the following:
+ * :code:`Iter` satisfies the C++ named requirement :code:`LegacyRandomAccessIterator`
```

Comment:
Unfortunately, `LegacyRandomAccessIterator` used at cppreference.com is not a thing in the standard.

In C++17, random access iterator requirements do not have a special name. In C++20 and beyond, the named requirement is called `Cpp17RandomAccessIterator`.

Reply:
I will take a renewed look at this in the context of `Cpp17RandomAccessIterator`.  
Assuming it is a good match for us still, is it reasonable to use `Cpp17RandomAccessIterator` within oneDPL's spec (given that oneDPL based upon C++17)?  
If not, should we try to distil down the key requirements we need and define our own baseline for a random access-like iterator?

Reply:
We do use the term "random access iterators" when describing the types of data which may be passed to oneDPL algorithms.  Should we just use that term here or should we try to define it more explicitly?

Reply:
Let's just use "random access iterator" for now.

Reply:
done.